crates/manifest/src/lib.rs

```diff
@@ -101,33 +112,38 @@ pub struct HttpTriggerConfiguration {
     /// Base path for the HTTP application.
     pub base: String,
 }
+
 impl Default for HttpTriggerConfiguration {
     fn default() -> Self {
         Self { base: "/".into() }
     }
 }
 
+impl TryFrom<ApplicationTrigger> for HttpTriggerConfiguration {
```

Comment:
Are these able to move into the trigger-specific crates at this point, or does that come later as part of the trigger executors work?

Reply:
They could be moved but they can't really be decoupled without more config refactoring

Reply:
I will keep them in the manifest for now